tests/integration/conftest.py

```diff
@@ -84,28 +83,30 @@ def app():
     )
 
     with app.app_context():
-        # Celery task imports must be local, otherwise their
-        # configuration would use the default pickle serializer.
-        from inspirehep.modules.migrator.tasks import migrate_from_file
+        with mock.patch('inspirehep.modules.records.receivers.index_new_cited_records.apply_async'):
```

Comment:
you can also do `with context_manager1, context_manager2:`, so you don't have to reindent everything.